app/controllers/emails_controller.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+
+# Controller that manages emails
+class EmailsController < ApiController
+  def index
+    render json: Email.all
+  end
+
+  def create
+    email = Email.create(email_params)
+    render json: email
+  end
```

Comment:
If you are moving this to be configurable in the backend, why doesn't it also exist an update? I assume that there is a good reason for this to be configurable this way, and not just have them in the code...

Reply:
The Active Admin has his own controller that have already an update function. I'm just using that create function to create the default templates in the seeds file

![EmailsAdminBackffice](https://user-images.githubusercontent.com/55494046/185368550-e74bd5ab-8c3e-4224-b070-df7cf71899e9.png)

Reply:
Oh, right! I remember now 😅